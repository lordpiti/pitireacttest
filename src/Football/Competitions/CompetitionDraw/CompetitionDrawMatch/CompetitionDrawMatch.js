import React from 'react';

const CompetitionDrawMatch = ( props ) => {

    return (
      <div class="mtch_container">
        <div class="match_unit">
          <div class="m_segment m_top winner" data-team-id="9">
              <span>
                  <a href="#">
                      <img width="25" height="25" src={props.match.localTeam.pictureLogo.url} alt={props.match.localTeam.name} />
                      <span class="truncate">{props.match.localTeam.name}</span>
                  </a>
                  <strong>{props.match.goalsLocal}</strong>
              </span>
          </div>
          <div class="m_segment m_botm loser" data-team-id="10">
              <span>
                  <a href="#">
                      <img width="25" height="25" src={props.match.visitorTeam.pictureLogo.url} alt={props.match.visitorTeam.name} />
                      <span class="truncate">{props.match.visitorTeam.name}</span>
                  </a>
                  <strong>{props.match.goalsVisitor}</strong>
              </span>
          </div>
          <div class="m_dtls">
              <a routerLink="/competitions/detail/{{competitionData.id}}/match/{{match.matchId}}">
                  <span>{props.match.date}</span>
              </a>        
          </div>
      </div>
  </div>
    )
};

export default CompetitionDrawMatch;