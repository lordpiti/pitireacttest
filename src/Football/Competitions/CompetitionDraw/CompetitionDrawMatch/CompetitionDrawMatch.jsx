import React from 'react';
import Formatters from '../../../utilities/formatters';

const CompetitionDrawMatch = ( props ) => {

    return (
      <div className="mtch_container">
        <div className="match_unit">
          <div className="m_segment m_top winner" data-team-id="9">
              <span>
                  <a href="#">
                      <img width="25" height="25" src={props.match.localTeam.pictureLogo.url} alt={props.match.localTeam.name} />
                      <span className="truncate">{props.match.localTeam.name}</span>
                  </a>
                  <strong>{props.match.goalsLocal}</strong>
              </span>
          </div>
          <div className="m_segment m_botm loser" data-team-id="10">
              <span>
                  <a href="#">
                      <img width="25" height="25" src={props.match.visitorTeam.pictureLogo.url} alt={props.match.visitorTeam.name} />
                      <span className="truncate">{props.match.visitorTeam.name}</span>
                  </a>
                  <strong>{props.match.goalsVisitor}</strong>
              </span>
          </div>
          <div className="m_dtls">
              <a routerLink="/competitions/detail/{{competitionData.id}}/match/{{match.matchId}}">
                  <span>{Formatters.formatDate(props.match.date)}</span>
              </a>        
          </div>
      </div>
  </div>
    )
};

export default CompetitionDrawMatch;