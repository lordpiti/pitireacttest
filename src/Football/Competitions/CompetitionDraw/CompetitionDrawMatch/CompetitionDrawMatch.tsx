import React from 'react';
import { Link } from 'react-router-dom';
import Formatters from '../../../utilities/formatters';

interface CompetitionDrawMatchProps {
  match: any;
  competitionData: any;
}

const CompetitionDrawMatch = (props: CompetitionDrawMatchProps) => {
  const { match, competitionData } = props;

  return (
    <div className='mtch_container'>
      <div className='match_unit'>
        <div className='m_segment m_top winner' data-team-id='9'>
          <span>
            <Link
              to={{
                pathname: `/teams/team-details/${match.localTeam.id}`,
              }}
            >
              <img
                width='25'
                height='25'
                src={match.localTeam.pictureLogo.url}
                alt={match.localTeam.name}
              />
              <span className='truncate'>{match.localTeam.name}</span>
            </Link>
            <strong>{match.goalsLocal}</strong>
          </span>
        </div>
        <div className='m_segment m_botm loser' data-team-id='10'>
          <span>
            <Link
              to={{
                pathname: `/teams/team-details/${match.visitorTeam.id}`,
              }}
            >
              <img
                width='25'
                height='25'
                src={match.visitorTeam.pictureLogo.url}
                alt={match.visitorTeam.name}
              />
              <span className='truncate'>{match.visitorTeam.name}</span>
            </Link>
            <strong>{match.goalsVisitor}</strong>
          </span>
        </div>
        <div className='m_dtls'>
          <Link
            to={{
              pathname: `/competitions/competition-details/${competitionData.id}/competition-rounds/match/${match.matchId}`,
            }}
          >
            <span>{Formatters.formatDate(match.date)}</span>
          </Link>
        </div>
      </div>
    </div>
  );
};

export default CompetitionDrawMatch;
