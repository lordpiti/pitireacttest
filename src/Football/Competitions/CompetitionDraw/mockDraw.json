{
  "eightLeft": [
    {
      "date": "2009-10-26T00:00:00",
      "stadium": {
        "id": 1,
        "name": "Estadio de Riazor",
        "address": null,
        "capacity": 0
      },
      "localTeam": {
        "id": 4,
        "playerList": null,
        "name": "Valencia C.F",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
          "url": "https://pititest.blob.core.windows.net/mycontainer/ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "visitorTeam": {
        "id": 9,
        "playerList": null,
        "name": "Almeria",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "1720.jpg",
          "url": "https://pititest.blob.core.windows.net/mycontainer/1720.jpg",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "goalsLocal": 0,
      "goalsVisitor": 0,
      "matchId": 764
    },
    {
      "date": "2009-07-02T00:00:00",
      "stadium": {
        "id": 1,
        "name": "Estadio de Riazor",
        "address": null,
        "capacity": 0
      },
      "localTeam": {
        "id": 7,
        "playerList": null,
        "name": "Málaga",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "5915d799-b58a-4ef6-87a0-cf45ed2cbe87-test.png",
          "url": "https://pititest.blob.core.windows.net/mycontainer/5915d799-b58a-4ef6-87a0-cf45ed2cbe87-test.png",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "visitorTeam": {
        "id": 14,
        "playerList": null,
        "name": "Osasuna",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "46be640c-1bdc-4472-a6dc-ebfc5df9687a-211.jpg",
          "url": "https://pititest.blob.core.windows.net/mycontainer/46be640c-1bdc-4472-a6dc-ebfc5df9687a-211.jpg",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "goalsLocal": 1,
      "goalsVisitor": 0,
      "matchId": 767
    },
    {
      "date": "2009-06-02T00:00:00",
      "stadium": {
        "id": 1,
        "name": "Estadio de Riazor",
        "address": null,
        "capacity": 0
      },
      "localTeam": {
        "id": 6,
        "playerList": null,
        "name": "Atlético de Madrid",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "2cd85554-3d56-4ade-824f-d6718dd0a057-test.png",
          "url": "https://pititest.blob.core.windows.net/mycontainer/2cd85554-3d56-4ade-824f-d6718dd0a057-test.png",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "visitorTeam": {
        "id": 11,
        "playerList": null,
        "name": "Sporting de Gijón",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "df217d75-e06a-4b7f-8089-4c49eb307c0d-test.png",
          "url": "https://pititest.blob.core.windows.net/mycontainer/df217d75-e06a-4b7f-8089-4c49eb307c0d-test.png",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "goalsLocal": 3,
      "goalsVisitor": 1,
      "matchId": 766
    },
    {
      "date": "2009-07-22T00:00:00",
      "stadium": {
        "id": 1,
        "name": "Estadio de Riazor",
        "address": null,
        "capacity": 0
      },
      "localTeam": {
        "id": 8,
        "playerList": null,
        "name": "Athletic de Bilbao",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "749.jpg",
          "url": "https://pititest.blob.core.windows.net/mycontainer/749.jpg",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "visitorTeam": {
        "id": 16,
        "playerList": null,
        "name": "Sevilla C.F",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "ce743178-43a7-47a0-b192-94caf6216695-test.png",
          "url": "https://pititest.blob.core.windows.net/mycontainer/ce743178-43a7-47a0-b192-94caf6216695-test.png",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "goalsLocal": 2,
      "goalsVisitor": 1,
      "matchId": 768
    }
  ],
  "eightRight": [
    {
      "date": "2009-05-18T00:00:00",
      "stadium": {
        "id": 1,
        "name": "Estadio de Riazor",
        "address": null,
        "capacity": 0
      },
      "localTeam": {
        "id": 5,
        "playerList": null,
        "name": "Villarreal",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "987.jpg",
          "url": "https://pititest.blob.core.windows.net/mycontainer/987.jpg",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "visitorTeam": {
        "id": 13,
        "playerList": null,
        "name": "Racing de Santander",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "b3ab87b1-5d7a-4c52-a7c7-71f33c90f3c9-1473.jpg",
          "url": "https://pititest.blob.core.windows.net/mycontainer/b3ab87b1-5d7a-4c52-a7c7-71f33c90f3c9-1473.jpg",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "goalsLocal": 3,
      "goalsVisitor": 0,
      "matchId": 765
    },
    {
      "date": "2009-10-22T00:00:00",
      "stadium": {
        "id": 1,
        "name": "Estadio de Riazor",
        "address": null,
        "capacity": 0
      },
      "localTeam": {
        "id": 1,
        "playerList": null,
        "name": "R.C Deportivo",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "f20b9256-8fb6-4d62-a177-286fc2e18e69-test.png",
          "url": "https://pititest.blob.core.windows.net/mycontainer/f20b9256-8fb6-4d62-a177-286fc2e18e69-test.png",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "visitorTeam": {
        "id": 12,
        "playerList": null,
        "name": "Numancia",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "1952.jpg",
          "url": "https://pititest.blob.core.windows.net/mycontainer/1952.jpg",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "goalsLocal": 1,
      "goalsVisitor": 1,
      "matchId": 761
    },
    {
      "date": "2009-06-26T00:00:00",
      "stadium": {
        "id": 1,
        "name": "Estadio de Riazor",
        "address": null,
        "capacity": 0
      },
      "localTeam": {
        "id": 2,
        "playerList": null,
        "name": "F.C Barcelona",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "07f2de74-b684-421b-a508-9bde3496aeed-test.png",
          "url": "https://pititest.blob.core.windows.net/mycontainer/07f2de74-b684-421b-a508-9bde3496aeed-test.png",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "visitorTeam": {
        "id": 10,
        "playerList": null,
        "name": "Valladolid",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "849.jpg",
          "url": "https://pititest.blob.core.windows.net/mycontainer/849.jpg",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "goalsLocal": 0,
      "goalsVisitor": 3,
      "matchId": 762
    },
    {
      "date": "2009-07-09T00:00:00",
      "stadium": {
        "id": 1,
        "name": "Estadio de Riazor",
        "address": null,
        "capacity": 0
      },
      "localTeam": {
        "id": 3,
        "playerList": null,
        "name": "Real Madrid",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "b111ec0e-3c8a-41e2-82ae-d7a8c8834a4a-test.png",
          "url": "https://pititest.blob.core.windows.net/mycontainer/b111ec0e-3c8a-41e2-82ae-d7a8c8834a4a-test.png",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "visitorTeam": {
        "id": 15,
        "playerList": null,
        "name": "Espanyol",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "afabcacb-9de5-483b-a1b3-e4b9702dac04-test.png",
          "url": "https://pititest.blob.core.windows.net/mycontainer/afabcacb-9de5-483b-a1b3-e4b9702dac04-test.png",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "goalsLocal": 1,
      "goalsVisitor": 0,
      "matchId": 763
    }
  ],
  "quarterFinalsLeft": [
    {
      "date": "2009-04-20T00:00:00",
      "stadium": {
        "id": 1,
        "name": "Estadio de Riazor",
        "address": null,
        "capacity": 0
      },
      "localTeam": {
        "id": 4,
        "playerList": null,
        "name": "Valencia C.F",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
          "url": "https://pititest.blob.core.windows.net/mycontainer/ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "visitorTeam": {
        "id": 14,
        "playerList": null,
        "name": "Osasuna",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "46be640c-1bdc-4472-a6dc-ebfc5df9687a-211.jpg",
          "url": "https://pititest.blob.core.windows.net/mycontainer/46be640c-1bdc-4472-a6dc-ebfc5df9687a-211.jpg",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "goalsLocal": 0,
      "goalsVisitor": 2,
      "matchId": 770
    },
    {
      "date": "2009-04-23T00:00:00",
      "stadium": {
        "id": 1,
        "name": "Estadio de Riazor",
        "address": null,
        "capacity": 0
      },
      "localTeam": {
        "id": 6,
        "playerList": null,
        "name": "Atlético de Madrid",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "2cd85554-3d56-4ade-824f-d6718dd0a057-test.png",
          "url": "https://pititest.blob.core.windows.net/mycontainer/2cd85554-3d56-4ade-824f-d6718dd0a057-test.png",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "visitorTeam": {
        "id": 16,
        "playerList": null,
        "name": "Sevilla C.F",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "ce743178-43a7-47a0-b192-94caf6216695-test.png",
          "url": "https://pititest.blob.core.windows.net/mycontainer/ce743178-43a7-47a0-b192-94caf6216695-test.png",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "goalsLocal": 2,
      "goalsVisitor": 2,
      "matchId": 772
    }
  ],
  "quarterFinalsRight": [
    {
      "date": "2009-03-12T00:00:00",
      "stadium": {
        "id": 1,
        "name": "Estadio de Riazor",
        "address": null,
        "capacity": 0
      },
      "localTeam": {
        "id": 5,
        "playerList": null,
        "name": "Villarreal",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "987.jpg",
          "url": "https://pititest.blob.core.windows.net/mycontainer/987.jpg",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "visitorTeam": {
        "id": 12,
        "playerList": null,
        "name": "Numancia",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "1952.jpg",
          "url": "https://pititest.blob.core.windows.net/mycontainer/1952.jpg",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "goalsLocal": 2,
      "goalsVisitor": 1,
      "matchId": 771
    },
    {
      "date": "2009-08-11T00:00:00",
      "stadium": {
        "id": 1,
        "name": "Estadio de Riazor",
        "address": null,
        "capacity": 0
      },
      "localTeam": {
        "id": 2,
        "playerList": null,
        "name": "F.C Barcelona",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "07f2de74-b684-421b-a508-9bde3496aeed-test.png",
          "url": "https://pititest.blob.core.windows.net/mycontainer/07f2de74-b684-421b-a508-9bde3496aeed-test.png",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "visitorTeam": {
        "id": 15,
        "playerList": null,
        "name": "Espanyol",
        "pictureLogo": {
          "id": 0,
          "bytes": null,
          "base64String": null,
          "fileName": "afabcacb-9de5-483b-a1b3-e4b9702dac04-test.png",
          "url": "https://pititest.blob.core.windows.net/mycontainer/afabcacb-9de5-483b-a1b3-e4b9702dac04-test.png",
          "containerReference": "mycontainer"
        },
        "stadium": null,
        "city": null
      },
      "goalsLocal": 1,
      "goalsVisitor": 1,
      "matchId": 769
    }
  ],
  "semifinalsLeft": {
    "date": "2009-08-13T00:00:00",
    "stadium": {
      "id": 1,
      "name": "Estadio de Riazor",
      "address": null,
      "capacity": 0
    },
    "localTeam": {
      "id": 4,
      "playerList": null,
      "name": "Valencia C.F",
      "pictureLogo": {
        "id": 0,
        "bytes": null,
        "base64String": null,
        "fileName": "ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
        "url": "https://pititest.blob.core.windows.net/mycontainer/ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
        "containerReference": "mycontainer"
      },
      "stadium": null,
      "city": null
    },
    "visitorTeam": {
      "id": 6,
      "playerList": null,
      "name": "Atlético de Madrid",
      "pictureLogo": {
        "id": 0,
        "bytes": null,
        "base64String": null,
        "fileName": "2cd85554-3d56-4ade-824f-d6718dd0a057-test.png",
        "url": "https://pititest.blob.core.windows.net/mycontainer/2cd85554-3d56-4ade-824f-d6718dd0a057-test.png",
        "containerReference": "mycontainer"
      },
      "stadium": null,
      "city": null
    },
    "goalsLocal": 3,
    "goalsVisitor": 3,
    "matchId": 773
  },
  "semifinalsRight": {
    "date": "2009-02-24T00:00:00",
    "stadium": {
      "id": 1,
      "name": "Estadio de Riazor",
      "address": null,
      "capacity": 0
    },
    "localTeam": {
      "id": 5,
      "playerList": null,
      "name": "Villarreal",
      "pictureLogo": {
        "id": 0,
        "bytes": null,
        "base64String": null,
        "fileName": "987.jpg",
        "url": "https://pititest.blob.core.windows.net/mycontainer/987.jpg",
        "containerReference": "mycontainer"
      },
      "stadium": null,
      "city": null
    },
    "visitorTeam": {
      "id": 15,
      "playerList": null,
      "name": "Espanyol",
      "pictureLogo": {
        "id": 0,
        "bytes": null,
        "base64String": null,
        "fileName": "afabcacb-9de5-483b-a1b3-e4b9702dac04-test.png",
        "url": "https://pititest.blob.core.windows.net/mycontainer/afabcacb-9de5-483b-a1b3-e4b9702dac04-test.png",
        "containerReference": "mycontainer"
      },
      "stadium": null,
      "city": null
    },
    "goalsLocal": 0,
    "goalsVisitor": 2,
    "matchId": 774
  },
  "final": {
    "date": "2009-05-20T00:00:00",
    "stadium": {
      "id": 1,
      "name": "Estadio de Riazor",
      "address": null,
      "capacity": 0
    },
    "localTeam": {
      "id": 4,
      "playerList": null,
      "name": "Valencia C.F",
      "pictureLogo": {
        "id": 0,
        "bytes": null,
        "base64String": null,
        "fileName": "ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
        "url": "https://pititest.blob.core.windows.net/mycontainer/ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png",
        "containerReference": "mycontainer"
      },
      "stadium": null,
      "city": null
    },
    "visitorTeam": {
      "id": 5,
      "playerList": null,
      "name": "Villarreal",
      "pictureLogo": {
        "id": 0,
        "bytes": null,
        "base64String": null,
        "fileName": "987.jpg",
        "url": "https://pititest.blob.core.windows.net/mycontainer/987.jpg",
        "containerReference": "mycontainer"
      },
      "stadium": null,
      "city": null
    },
    "goalsLocal": 0,
    "goalsVisitor": 2,
    "matchId": 775
  }
}
