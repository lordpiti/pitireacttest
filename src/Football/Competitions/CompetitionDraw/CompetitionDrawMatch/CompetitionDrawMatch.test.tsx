import React from 'react';
import CompetitionDrawMatch from './CompetitionDrawMatch';
import { shallow } from 'enzyme';
import mockDraw from '.././mockDraw.json';

const competitionData = { id: 1 };
const matchData = mockDraw.eightLeft[0];

describe('CompetitionDraw', () => {
  it(`should render correctly`, () => {
    const wrapper = shallow(
      <CompetitionDrawMatch
        competitionData={competitionData}
        match={matchData}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
