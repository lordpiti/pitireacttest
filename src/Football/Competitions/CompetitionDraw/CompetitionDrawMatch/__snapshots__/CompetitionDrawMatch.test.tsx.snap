// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompetitionDraw should render correctly 1`] = `
<div
  className="mtch_container"
>
  <div
    className="match_unit"
  >
    <div
      className="m_segment m_top winner"
      data-team-id="9"
    >
      <span>
        <Link
          to={
            Object {
              "pathname": "/teams/team-details/4",
            }
          }
        >
          <img
            alt="Valencia C.F"
            height="25"
            src="https://pititest.blob.core.windows.net/mycontainer/ab79f4b3-4d2d-4380-9596-2f7bd3f9012e-test.png"
            width="25"
          />
          <span
            className="truncate"
          >
            Valencia C.F
          </span>
        </Link>
        <strong>
          0
        </strong>
      </span>
    </div>
    <div
      className="m_segment m_botm loser"
      data-team-id="10"
    >
      <span>
        <Link
          to={
            Object {
              "pathname": "/teams/team-details/9",
            }
          }
        >
          <img
            alt="Almeria"
            height="25"
            src="https://pititest.blob.core.windows.net/mycontainer/1720.jpg"
            width="25"
          />
          <span
            className="truncate"
          >
            Almeria
          </span>
        </Link>
        <strong>
          0
        </strong>
      </span>
    </div>
    <div
      className="m_dtls"
    >
      <Link
        to={
          Object {
            "pathname": "/competitions/competition-details/1/competition-rounds/match/764",
          }
        }
      >
        <span>
          October 26, 2009
        </span>
      </Link>
    </div>
  </div>
</div>
`;
